sdk/tests/client/transactions.rs

```diff
@@ -68,3 +69,69 @@ async fn send_basic_output() -> Result<()> {
 
     Ok(())
 }
+
+#[ignore]
+#[tokio::test]
+async fn custom_input() -> Result<()> {
+    let (client, secret_manager) = create_client_and_secret_manager_with_funds(None).await?;
+
+    let token_supply = client.get_token_supply().await?;
+
+    let address = secret_manager
+        .generate_ed25519_addresses(GetAddressesOptions::from_client(&client).await?.with_range(0..1))
+        .await?[0];
+
+    let bech32_hrp = client.get_bech32_hrp().await?;
+
+    // First create multiple outputs on our address, so there is even the possibility to select another input
+    let output = BasicOutputBuilder::new_with_amount(1_000_000)
+        .add_unlock_condition(AddressUnlockCondition::new(address))
+        .finish_output(token_supply)?;
+    let block = client
+        .block()
```

Comment:
Unrelated to this PR, but I feel like this should be `build_block()`, right??

Reply:
We already had `send_transaction_message, send_indexation_message`, `send`, `message`,  `block` now :trollface:
The thing is that it's not only building it, but also sending
Also in 2.0 we most probably remove this completely (the transaction part there is already removed), so I would just keep it as it is now

Reply:
True, I don't care that much how we name it, open an issue if you want